case_studies/sdss_galaxies_vae/reconstruction.py

```diff
@@ -398,7 +398,7 @@ def create_figure_at_point(
 
     if est_catalog is not None:
         tile_est_catalog = est_catalog.to_tile_params(
-            tile_map_cropped.tile_slen, tile_map_cropped.max_sources
+            tile_map_cropped.tile_slen, tile_map_cropped.max_sources, ignore_extra_sources=True
```

Comment:
Hmm the `est_catalog` (is it from BLISS?) cannot possible throw this error. Need to investigate